src/main/java/Duke.java

```diff
@@ -1,10 +1,176 @@
+        System.out.println("  " + t);
+        System.out.println("You now have " + Task.getTaskCount() + " task(s) in your list.");
+    }
+
+    public static void done(Task[] tasks, int taskNumber) {
+        System.out.println("Yay! Pepepopo has marked your task as done:");
+        System.out.println(tasks[taskNumber]);
+        divider();
+    }
+
+    public static String getInput() {
+        String line;
+        Scanner in = new Scanner(System.in);
+
+        line = in.nextLine();
+        return line;
+    }
+    public static void executeCommand(String input) {
+        String[] commandAndParams = splitString(input, " ");
+        String command = commandAndParams[0];
+        String params = commandAndParams[1];
+        switch (command) {
+        case COMMAND_TODO:
+            executeTodo(params);
+            break;
+        case COMMAND_DEADLINE:
+            executeDeadline(params);
+            break;
+        case COMMAND_EVENT:
+            executeEvent(params);
+            break;
+        case COMMAND_DONE:
+            executeDone(params);
+            break;
+        case COMMAND_LIST:
+            executeList();
+            break;
+        case COMMAND_BYE:
+            executeBye();
+            break;
+        default:
+            break;
+        }
+    }
+
+    public static void addTask(Task t) {
+        tasks[Task.getTaskCount()] = t;
+        Task.setTaskCount();
+        added(t);
+        divider();
+    }
+
+    public static void executeTodo(String params) {
+        Todo t = new Todo(params);
+        addTask(t);
+    }
+
+    public static void executeDeadline(String params) {
+        String[] descAndBy = splitString(params, DEADLINE_BY_PREFIX);
+        String description = descAndBy[0];
+        String by = descAndBy[1];
+        Deadline d = new Deadline(description, by);
+        addTask(d);
+    }
+
+    public static void executeEvent(String params) {
+        String[] descAndAt = splitString(params, EVENT_AT_PREFIX);
+        String description = descAndAt[0];
+        String at = descAndAt[1];
+        Event e = new Event(description, at);
+        addTask(e);
+    }
+
+    public static void executeDone(String params) {
+        if (isDone(params)) {
+            int taskNumber = Integer.parseInt(params) - 1;
+            tasks[taskNumber].setDone();
+            done(tasks, taskNumber);
+        }
+    }
+
+    public static void executeList() {
+        Task[] listPrint = Arrays.copyOf(tasks, Task.getTaskCount());
+        int curr = 1;
+        for (Task item : listPrint) {
+            System.out.println(curr + "." + item);
+            curr += 1;
+        }
+        divider();
+    }
+
+    public static void executeBye() {
+        bye();
+        System.exit(0);
+    }
+
+    public static String[] splitString(String input, String separator) {
+        String[] split = input.trim().split(separator, 2);
+        return split.length == 2 ? split : new String[] {split[0], ""};
```

Comment:
Good use of ternary conditions but I think you could split the { } or use if else here. It took me a while to understand what this line was doing :( You can also take out the 2 as a constant and name it something else because I am not too sure what 2 represents in this function.